### DISCUSSION TITLE
Stromverbrauch anzeigen lassen?

### DISCUSSION THREAD

**Discussion by mrmatze007** [brutella/hkknx-public#209] [General]
Hi zusammen,

gerne würde ich nach dem Einbinden der KNX-Geräte in HomeKit ein Dashboard anlegen. Nun gibt es iOS-Dashboards, die in der Lage sind, den Stromverbrauch von Geräten anzuzeigen (= Basis sind die Eve Zwischenstecker mit Stromzähler, die dann ausgelesen werden).

Daher habe ich folgende Frage: Ist es mit hkknx möglich, den Stromverbrauch der KNX-Aktorik an HomeKit zu übergeben, sodass dieser ins Dashboard eingebunden werden kann?

Danke euch für für den Support.

Grüße,
mrmatze007

**Comment by chef76** [comment]
Ob die Übergabe an Homekit geht weiß ich nicht, aber ich nutze dafür ein Dashboard der Brücke:

https://hochgatterer.me/de/hkknx/docs/scripts/dashboard/

**Comment by brutella** [comment]
HomeKit hat für den Stromverbrauch keinen Typ definiert. Nur Drittanbieter-Apps (wie Eve oder [Home+](https://itunes.apple.com/us/app/home-smart-home-automation/id995994352)) zeigen dir diesen Typ an.

Ich möchte aber vermeiden, dass man Drittanbieter-Apps verwenden muss, um alle Funktionen von hkknx verwenden zu können. Alles sollte mit der Apple Home-App funktionieren.
